### ISSUE TITLE
shellcheck site doesn't recognize shellcheck shell directive?

### ISSUE THREAD

**Issue by matthewpersico** [koalaman/shellcheck#1176]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2148
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [x] It's ONLY reproducible on shellcheck.net.

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
# sheckcheck shell=bash
declare opt_v=''
declare v=''

foobar $opt_v $v

```

#### Here's what shellcheck currently says:
```
Line 1	SC2148: Tips depend on target shell and yours is unknown. Add a shebang.
```
#### Here's what I wanted or expected to see:

Nothing.  I expected the directive to eliminate the error shown. The shellcheck PROGRAM handles this just fine - it's the site that has the problem.

**Comment by per-bohlin** [comment]
Your shebang looks misspelled. "she**ck**check"

**Comment by matthewpersico** [comment]
https://www.bing.com/videos/search?q=never+mind+gilda+radner&&view=detail&mid=2EF9E6A096321B858CBA2EF9E6A096321B858CBA&rvsmid=56401A1E561B6B35209C56401A1E561B6B35209C&FORM=VDQVAP
